0.6 - Updated to bevy 0.10.1

BlackPhlox opened the pull request.

leeola commented:
Looks like this PR is old, as we're now on 0.10. Were there blockers to merging this? I'm looking into contributing if possible, as i need this working on 0.10

BlackPhlox commented:
@leeola Thanks for notifying me! I've updated this branch to bevy 0.10.1 and I've done some rudimentary testing, if you find any bugs or any API changes you would like to see, please let me know, and you're very welcome to contribute!
We also got a couple of open https://github.com/BlackPhlox/bevy_midi/issues and if you have any ideas or PRs I will gladly review them! 😊

BlackPhlox commented:
Merged and updated an old Egui example branch and added a prelude.
I've tested the current code and seems to be a point where I feel confident in creating a new release 🚀
